x-pack/plugins/security_solution/public/resolver/view/panels/node_list.tsx

```diff
@@ -132,42 +99,26 @@ export const NodeList = memo(() => {
     []
   );
 
-  const { processNodePositions } = useSelector(selectors.layout);
-  const nodeHrefs: Map<SafeResolverEvent, string | null | undefined> = useSelector(
-    (state: ResolverState) => {
-      const relativeHref = selectors.relativeHref(state);
-      return new Map(
-        [...processNodePositions.keys()].map((processEvent) => {
-          const nodeID = event.entityIDSafeVersion(processEvent);
-          if (nodeID === undefined) {
-            return [processEvent, null];
-          }
-          return [
-            processEvent,
-            relativeHref({
-              panelView: 'nodeDetail',
-              panelParameters: {
-                nodeID,
-              },
-            }),
-          ];
-        })
-      );
-    }
-  );
-  const processTableView: ProcessTableView[] = useMemo(
-    () =>
-      [...processNodePositions.keys()].map((processEvent) => {
-        const name = event.processNameSafeVersion(processEvent);
-        return {
-          name,
-          timestamp: event.timestampAsDateSafeVersion(processEvent),
-          event: processEvent,
-          href: nodeHrefs.get(processEvent) ?? undefined,
-        };
-      }),
-    [processNodePositions, nodeHrefs]
+  const processTableView: ProcessTableView[] = useSelector(
+    useCallback((state: ResolverState) => {
```

Comment:
The `useSelector`/`useCallback` pattern gets a value that's based on state and is memoized.

Reply:
that's interesting

Reply:
Really cool